docs/channels/category-heading.md

```diff
@@ -30,6 +30,20 @@ If no Categories are being shown, the tag will not show anything contained withi
 
 [TOC=3]
 
+### `category_id=`
+
+    category_id="5"
+
+The category_id= parameter allows you to override the detection of current category on URL.
+
+### `category_url_title=`
+
+    category_url_title="press-releases"
+
+The category_id= parameter allows you to override the detection of current category on URL without. Also, you don't need to use the [Category URL Titles In Links](control-panel/settings/content-design.md) feature.
+
+NOTE: **Note:** Unless you are using the `relaxed_categories` parameter, you can only list one channel name, since each channel can have separate category groups.
```

Comment:
> you can only list one channel name
There are no other mentions of channel name, so I'm not sure what this refers to